### PR TITLE
Add Node.js support

### LINKED ISSUE #25: Support Node.js
This package will eventually need to be distributed to our many Node.js users, so it should support running on Node.js as well as the Dart VM.

### PR DESCRIPTION
**Pull request by nex3**
Closes #25

### CODE REVIEW

**Comment by nex3** [comment]
Note: this isn't ready for review yet, I'm still figuring out how to set up the Travis config properly.

**Comment by nex3** [comment]
Okay, should be good to review now!

File: tool/grind/utils.dart [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// Copyright 2018 Google Inc. Use of this source code is governed by an
+// MIT-style license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import 'dart:io';
+
+import 'package:pub_semver/pub_semver.dart';
+import 'package:yaml/yaml.dart';
+
+/// The version of Dart Sass.
```
Comment (jathak) [inline tool/grind/utils.dart]: This is the migrator's version, not Dart Sass's

Comment (nex3) [reply tool/grind/utils.dart]: Done.

**Pull request merged**